inlang/source-code/sdk2/src/project/logic/initHandleSaveToLixOnChange.ts

```diff
@@ -16,12 +16,16 @@ export async function initHandleSaveToLixOnChange(args: {
 		arity: 0,
 		// @ts-expect-error - dynamic function
 		xFunc: () => {
-			const data = contentFromDatabase(args.sqlite);
-			args.lix.db
-				.updateTable("file")
-				.set("data", data)
-				.where("path", "=", "/db.sqlite")
-				.execute();
+			(async () => {
+				await new Promise((resolve) => setTimeout(resolve, 100));
```

Comment:
@samuelstroschein  could you comment why this timeout is required?

Reply:
See https://github.com/opral/monorepo/pull/3059#discussion_r1714438603. I don't know why. It seems like the AFTER triggers are triggering before the transaction went through, leading to no diffs.